src/systems/imu/Imu.cc

```diff
@@ -82,6 +86,19 @@ Imu::Imu() : System(), dataPtr(std::make_unique<ImuPrivate>())
 //////////////////////////////////////////////////
 Imu::~Imu() = default;
 
+//////////////////////////////////////////////////
+void Imu::Configure(const Entity & /*_entity*/,
+    const std::shared_ptr<const sdf::Element> &_sdf,
+    gazebo::EntityComponentManager & /*_ecm*/,
+    gazebo::EventManager & /*_eventMgr*/)
+{
+  if (_sdf->HasElement("output_orientation"))
```

Comment:
This would apply to all IMUs in simulation. Have you considered providing more granular configuration?

Reply:
updated changes in this PR to parse a new `enable_orientation` SDF element (https://github.com/ignitionrobotics/sdformat/pull/651) that allows each imu sensor to enable/disable orientation output. 48e171f